support-frontend/assets/helpers/abTests/abtest.ts

```diff
@@ -97,7 +97,7 @@ function getParticipationsFromUrl(): Participations | null | undefined {
 }
 
 function getServerSideParticipations(): Participations | null | undefined {
-	if (window && window.guardian && window.guardian.serversideTests) {
+	if (!(Object.keys(window.guardian.serversideTests).length === 0)) {
```

Comment:
As `window.guardian.serversideTests` is an object checking for the presence of keys here as an empty object is truthy